node-runner-cli/Pipfile

```diff
@@ -4,18 +4,38 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true, ref = "v1.1.1"}
+certifi = "==2023.7.22"
 requests = "==2.26.0"
-PyYAML = "==6.0.0"
 deepmerge = "==0.3.0"
 bech32 = "==1.2.0"
 ecdsa = "==0.17.0"
 cryptography="==36.0.1"
 deepdiff="==5.8.0"
-Jinja2="==3.1.2"
 boto3="==1.20.33"
 botocore="==1.23.33"
 pytest="==7.4.0"
+cffi = "==1.15.1"
+charset-normalizer = "==2.0.12"
+exceptiongroup = "==1.1.2"
+idna = "==3.4"
+importlib-metadata = "==6.7.0"
+iniconfig = "==2.0.0"
+jinja2 = "==3.1.2"
+jmespath = "==0.10.0"
```

Comment:
Are these main dependencies or transitive dependencies?  We use pip file for main dependencies from which we capture the requirements.txt with all the dependencies

Reply:
so the Pipfile here is the result of doing `pipenv install -r requirements.txt` which basically generates the Pipfile for the requirements.txt file. This is because snyk is failing because we certifi needs to be updated. I updated it in the requirements.txt file and it kept failing. That's when I found that snyk uses the Pipfile. So this is basically an attempt to have them both in the same page

Reply:
https://pipenv.pypa.io/en/latest/pipfile/#importing-from-requirements-txt